### PR TITLE
Oma error optimization

### PR DESCRIPTION
**Pull request by khickey-newrelic**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Enhances the error optimization document with some improved language, screenshots, and links.  All screenshots come from NR demo accounts

### CODE REVIEW

**Comment by bradleycamacho** [comment]
Hi @khickey-newrelic could you sign the CLA above?

**Comment by khickey-newrelic** [comment]
@bradleycamacho CLA signed.

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx [hunk 1]
```diff
@@ -245,15 +371,15 @@ If neither of the above statuses is suitable, assign the error to the appropriat
...
+As the velocity of errors coming into errors inbox goes down, a regular team session may no longer be a good use of time. An alternative is to [link Errors inbox to Slack](/docs/errors-inbox/error-external-services#slack) and either a) designate a rotating person to monitor the channel and resolve/ignore/assign error groups as they come in or b) allow the team to proactively respond to error groups.
 
 ### Use CodeStream [#codestream]
-Many of your error groups are going to require code changes to resolve. [Integrating CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) allows you to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 
+Many of your error groups are going to require code changes to resolve. [Integrate CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) to allow you to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx:380]:
```suggestion
Many of your error groups are going to require code changes to resolve. [Use CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 
```

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx [hunk 2]
```diff
@@ -245,15 +371,15 @@ If neither of the above statuses is suitable, assign the error to the appropriat
...
+Many of your error groups are going to require code changes to resolve. [Integrate CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) to allow you to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 
 
-Integrating CodeStream with errors inbox also gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
+The CodeStream integration gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx:382]:
```suggestion
New Relic with CodeStream gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
```

**Review by bradleycamacho — DISMISSED** [review]
@paperclypse could you take a quick look at my two suggestions and give me your thoughts? Thanks!

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx:380]: What do you think about: "Connect CodeStream to your New Relic account so you can open the offending..."

Also, "offending" is so funny :D

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx:382]: I think replace "The CodeStream integration" with "New Relic with CodeStream" or even just "CodeStream" is the right call.

**Comment by khickey-newrelic** [comment]
Hey @paperclypse those changes are/would have been fine. 

And yes, 'offending' is funny.. What can I say this stuff brings out the developer in me. Be passionate or go home. :P

**Pull request merged**